File: java/test/Ice/inheritance/CBI.java
// **********************************************************************
//
// Copyright (c) 2003-2007 ZeroC, Inc. All rights reserved.
//
// This copy of Ice is licensed to you under the terms described in the
// ICE_LICENSE file included in this distribution.
//
// **********************************************************************

public final class CBI extends Test.MB.CB
{
    public
    CBI()
    {
    }

    public Test.MA.CAPrx
    caop(Test.MA.CAPrx p, Ice.Current current)
    {
        return p;
    }

    public Test.MB.CBPrx
    cbop(Test.MB.CBPrx p, Ice.Current current)
    {
        return p;
    }
}

File: cs/test/Ice/inheritance/CBI.cs
// **********************************************************************
//
// Copyright (c) 2003
// ZeroC, Inc.
// Billerica, MA, USA
//
// All Rights Reserved.
//
// Ice is free software; you can redistribute it and/or modify it under
// the terms of the GNU General Public License version 2 as published by
// the Free Software Foundation.
//
// **********************************************************************

public sealed class CBI : MB.CB
{
    public CBI()
    {
    }
    
    public override MA.CAPrx caop(MA.CAPrx p, Ice.Current current)
    {
        return p;
    }
    
    public override MB.CBPrx cbop(MB.CBPrx p, Ice.Current current)
    {
        return p;
    }
}
